frontend/pages/group/data/foods.vue

```diff
@@ -87,6 +87,11 @@
             :label="$t('data-pages.foods.food-label')"
           >
           </v-autocomplete>
+          <v-checkbox
+            v-model="createTarget.onHand"
+            :label="$t('data-pages.foods.on-hand-checkbox-label')"
```

Comment:
Can we give this a simple "On Hand" label and then add a caption? Something like:

```html
<v-checkbox
  v-model="createTarget.onHand"
  hide-details
  :label="$tc('tool.on-hand')"
/>
<p class="text-caption mt-1">
  {{ $tc("data-pages.foods.on-hand-checkbox-label") }}
</p>
```